server.js

```diff
@@ -1,14 +1,14 @@
 var express = require('express');
```

Comment:
I changed the package.json to start from server/server,js so this no longer needs to be moved to the root.